diff -ru automake-1.11.old/automake.in automake-1.11/automake.in
--- automake-1.11.old/automake.in	2018-05-15 12:43:25.011964242 +0200
+++ automake-1.11/automake.in	2018-05-15 12:44:20.724619299 +0200
@@ -4110,7 +4110,7 @@
 sub substitute_ac_subst_variables ($)
 {
   my ($text) = @_;
-  $text =~ s/\${([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
+  $text =~ s/\$[{]([^ \t=:+{}]+)}/substitute_ac_subst_variables_worker ($1)/ge;
   return $text;
 }
 
